Sources/BrowserServicesKit/Autofill/AutofillUserScript+SourceProvider.swift

```diff
@@ -33,23 +33,27 @@ public class DefaultAutofillSourceProvider: AutofillUserScriptSourceProvider {
 
     let privacyConfigurationManager: PrivacyConfigurationManaging
     let properties: ContentScopeProperties
+    let isDebug: Bool
```

Comment:
One Thought I has was whether to remove this as a property and instead pass it as a parameter to the `loadJS` method. However, it doesn’t make a large difference, and the way you have it here means there is symmetry between the builder and class. Just wanted to share my thought process. Looks good!

Reply:
Good shout and have made that change, thanks!